test/sqllogictest/transform/aggregation_nullability.slt

```diff
@@ -280,25 +280,24 @@ query T multiline
 EXPLAIN WITH(arity, join implementations) select t1.f1, sum(t2.f1), max(t2.f1) from t1 LEFT JOIN t2 ON t1.f1 = t2.f1 group by t1.f1 having max(t2.f1) >= 0;
 ----
 Explained Query:
-  Filter (#2 >= 0) // { arity: 3 }
```

Comment:
This surprised me the first time, but what's happened is that we've concluded that `#0 == #2`, which allows us to rewrite this predicate as `#0 >= 0` and then push it down through the `Reduce`.

Reply:
Nice!